src/main/java/net/dancier/dancer/core/ProfileController.java

```diff
@@ -40,7 +42,7 @@ public ResponseEntity<ProfileDto> get(@CurrentUser AuthenticatedUser authenticat
 
     @Secured(ROLE_USER)
     @PutMapping
-    public ResponseEntity put(@CurrentUser AuthenticatedUser authenticatedUser, @RequestBody ProfileDto profileDto) {
+    public ResponseEntity put(@CurrentUser AuthenticatedUser authenticatedUser, @Valid @RequestBody ProfileDto profileDto) {
```

Comment:
What do you expect that this validation Annotation will do?
I did not see any ValidationConstraints in the ProfileDto?